src/action-types.js

```diff
@@ -1,3 +1,4 @@
+// @flow
```

Comment:
It at least infers their types, although it might be that consumers of this file that have the `// @flow` declaration do this anyway.